versioneer.py

```diff
@@ -464,9 +463,10 @@ def run_command(commands, args, cwd=None, verbose=False, hide_stderr=False):
         if verbose:
             print("unable to find command, tried %s" % (commands,))
         return None
+
     stdout = p.communicate()[0].strip()
-    if sys.version_info[0] >= 3:
-        stdout = stdout.decode()
+    stdout = stdout.decode()
```

Comment:
`stdout = p.communicate()[0].strip().decode()` is probably better to avoid mypy assigment of string to bytes errors if types are added.

Reply:
A bit out of scope for this PR, but sure, why not.